src/OrbitGl/TrackManager.cpp

```diff
@@ -317,7 +331,7 @@ int TrackManager::FindMovingTrackIndex() {
 
 void TrackManager::UpdateTrackPrimitives(Batcher* batcher, uint64_t min_tick, uint64_t max_tick,
                                          PickingMode picking_mode) {
-  for (auto& track : visible_tracks_) {
+  for (auto& track : GetTracksOnScreen()) {
```

Comment:
Don't use auto for simple types: https://google.github.io/styleguide/cppguide.html#Type_deduction